analysis/configuration/configs.py

```diff
@@ -6,6 +6,7 @@ class PercentChange:
     contrib_to_overall_change_threshold_percent: int = attr.ib()
     dimensions: list = attr.ib()
     sort_by: list = attr.ib()
+    overall_threshold_percent: float = attr.ib()
```

Comment:
As we add more config options I wonder if some of them could be optional and with default values.